python/foglamp/services/common/microservice.py

```diff
@@ -50,15 +50,25 @@ class FoglampMicroservice(FoglampProcess):
     _protocol = "http"
     """ communication protocol """
 
-    def __init__(self, default_config):
+    def __init__(self):
         super().__init__()
         try:
+            # Configuration handled through the Configuration Manager
+            default_config = {
+                'local_services': {
+                    'description': 'Restrict microservice to localhost',
+                    'type': 'string',
```

Comment:
boolean

default value False

so default will bind to 0.0.0.0

Reply:
Missed that before. Done.